### PR TITLE
ENG-3673 modified mysql yaml default configuration

### PR DESCRIPTION
**Pull request by jeffgo10**
configuration is based from default yaml configuration generated from generator-jhipster-entando

### CODE REVIEW

File: resources/svc/default-mysql.yml [hunk 1]
```diff
@@ -1,18 +1,16 @@
-version: '3.1'
+# This configuration is intended for development purpose, it's **your** responsibility to harden it for production
+version: '3.8'
 
 services:
   mysql:
-    image: mysql
-    restart: always
+    image: mysql:8.0.26
```
Comment (alepintus) [inline resources/svc/default-mysql.yml:6]: Hi @jeffgo10 , The image version could be moved to package.json and added here as variable. @cecchisandrone WDYT?

File: resources/svc/default-mysql.yml [hunk 2]
```diff
@@ -1,18 +1,16 @@
-version: '3.1'
+# This configuration is intended for development purpose, it's **your** responsibility to harden it for production
+version: '3.8'
```
Comment (cecchisandrone) [inline resources/svc/default-mysql.yml]: let's use 3.7 as 3.8 is too recent

File: resources/svc/default-mysql.yml [hunk 3]
```diff
@@ -1,18 +1,16 @@
...
 
 services:
   mysql:
-    image: mysql
-    restart: always
+    image: mysql:8.0.26
+    # volumes:
+    #   - ~/volumes/entando-bundle-cli/%BUNDLENAME%/mysql/:/var/lib/mysql/
```
Comment (cecchisandrone) [inline resources/svc/default-mysql.yml:9]: what's this?

**Review by alepintus — CHANGES_REQUESTED** [review]

File: resources/svc/default-mysql.yml (hunk 3 above)
Comment (jeffgo10) [reply resources/svc/default-mysql.yml:9]: @cecchisandrone this is an option to externalize your data, but I just copied that exact comment from the generated yaml from `generator-jhipster-entando`. Could be a quick option on externalizing data. do you want to remove this?

File: resources/svc/default-mysql.yml (hunk 1 above)
Comment (cecchisandrone) [reply resources/svc/default-mysql.yml:6]: We may do this to have the same approach as keycloak. In any case this is not a custom image.

Comment (jeffgo10) [reply resources/svc/default-mysql.yml:6]: @alepintus i'm up for it. should we implement this @cecchisandrone ?

File: resources/svc/default-mysql.yml (hunk 3 above)
Comment (cecchisandrone) [reply resources/svc/default-mysql.yml:9]: If you want to keep this, put a comment explaining it

File: resources/svc/default-mysql.yml (hunk 1 above)
Comment (cecchisandrone) [reply resources/svc/default-mysql.yml:6]: @jeff keep it as it is. @alepintus will do the version centralization in the scope of ENG-3672

File: resources/svc/default-mysql.yml (hunk 3 above)
Comment (jeffgo10) [reply resources/svc/default-mysql.yml:9]: done

File: resources/svc/default-mysql.yml (hunk 2 above)
Comment (jeffgo10) [reply resources/svc/default-mysql.yml]: done

File: resources/svc/default-mysql.yml [hunk 4]
```diff
@@ -1,18 +1,17 @@
...
-    image: mysql
-    restart: always
+    image: mysql:8.0.26
+    # If you want to relocate the mysql data (volume) to your local directory, uncomment the lines below
+    # volumes:
+    #   - ~/volumes/entando-bundle-cli/%BUNDLENAME%/mysql/:/var/lib/mysql/
     environment:
-      - MYSQL_DATABASE=%BUNDLENAME%-db
-      - MYSQL_ROOT_PASSWORD=adminadmin
+      - MYSQL_ALLOW_EMPTY_PASSWORD=yes
```
Comment (cecchisandrone) [inline resources/svc/default-mysql.yml]: Empty password? Let's use `%BUNDLENAME%-user` and `%BUNDLENAME%-password`, same for postgres

Comment (jeffgo10) [reply resources/svc/default-mysql.yml]: done. I added MYSQL_RANDOM_ROOT_PASSWORD as root password is needed in order the service to work

**Review by cecchisandrone — APPROVED** [review]

**Pull request merged**